### PR TITLE
OTC-70: RFC  103  Report Overview of Commissions

### PR DESCRIPTION
**Pull request by druchniewicz**
Related ticket: https://openimis.atlassian.net/browse/OTC-70

### CODE REVIEW

File: IMIS/Reports.aspx.vb [hunk 1]
```diff
@@ -538,6 +540,13 @@ Partial Public Class Reports
         ddlScope.DataValueField = "ScopeID"
         ddlScope.DataBind()
     End Sub
+
+    Private Sub FillReportScope()
+        scopesList.DataSource = reports.GetReportScope()
```
Comment (dragos-dobre) [inline IMIS/Reports.aspx.vb:545]: There is already the ddlScope select. I would suggest change the name or use the same ddlScope select for this report.

File: IMIS/Reports.aspx.vb [hunk 2]
```diff
@@ -1333,19 +1343,33 @@ Partial Public Class Reports
 
         Dim sSubTitle As String = ""
 
+        If Val(scopesList.SelectedValue) = 0 Then
+            IMIS_EN.eReports.Scope = 0
```
Comment (dragos-dobre) [inline IMIS/Reports.aspx.vb:1347]: Can't this be `IMIS_EN.eReports.Scope=Val(scopesList.SelectedValue)` and remove the 2 ifs?

File: IMIS/Reports.aspx.vb [hunk 3]
```diff
@@ -1333,19 +1343,33 @@ Partial Public Class Reports
...
+        ElseIf Val(scopesList.SelectedValue) = 1 Then
+            IMIS_EN.eReports.Scope = 1
+        End If
 
-        If Val(ddlEnrolmentOfficer.SelectedValue) = 0 And Val(ddlProduct.SelectedValue) = 0 Then
+        If Val(ddlMode.SelectedValue) = 0 Then
+            IMIS_EN.eReports.Mode = 0
```
Comment (dragos-dobre) [inline IMIS/Reports.aspx.vb:1353]: Can't this be `IMIS_EN.eReports.Mode=Val(ddlMode.SelectedValue)` and remove the 2 ifs?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Review by dragos-dobre — APPROVED** [review]

**Comment by dragos-dobre** [comment]
@druchniewicz Can you solve the conflicting files?

**Comment by druchniewicz** [comment]
resolved

**Pull request merged**